### PR TITLE
Optimize output

### PR DESCRIPTION
**Pull request by dimonomid**
Remove workarounds added while implementing --wrap, so that redundant
trailing sequences like `\x1b[32m\x1b[0m` are removed.

Also, actually run wrap regression tests; it should have been done in
the --wrap commit but somehow slipped away while addressing PR comments.

---
This change is [Reviewable](https://reviewable.io/reviews/ymattw/cdiff/74)

### CODE REVIEW

File: tests/diff-ru/out.side-by-side [hunk 1]
```diff
@@ -15,7 +15,7 @@
 [0m[33m--- a/common/foo.txt	2013-02-22 20:28:32.000000000 +0800
 [0m[33m+++ b/common/foo.txt	2013-02-22 20:30:18.000000000 +0800
 [0m[1;34m@@ -1 +1 @@
-[0m[33m1[0m [31m[7m[31mHello[0m[31m[0m                                                                            [0m[33m1[0m [32m[7m[32mworld[0m[32m[0m
+[0m[33m1[0m [31m[7m[31mHello[0m                                                                            [0m[33m1[0m [32m[7m[32mworld[0m
 [36mOnly in b: date.txt
 [0m[33m[0m[33m[0m[36mOnly in a: time.txt
-[0m[33m[0m[33m[0m
+[0m[33m[0m[33m[0m
```
Comment (ymattw) [inline tests/diff-ru/out.side-by-side]: I noticed a regression here: EOL at line end disappeared. If input file has the char, output should keep it too.

**Comment by ymattw** [comment]
Thanks again, please double check if there's a easy fix for the EOL problem.

**Comment by ymattw** [comment]
I figured out it was there from earlier revisions, just a special case when old_path and new_path are not present (e.g. "Only in ..." case). Somehow the test data was changed on commit time. So this was my fault :).

**Comment by dimonomid** [comment]
Cool.

Yeah sorry I was unable to dedicate time for that debugging. Thanks for debugging by yourself. :)

**Pull request merged**